apps/mobile-alphabet/src/app/Components/alphabet-card-detail.screen.tsx

```diff
@@ -132,10 +132,16 @@ export function AlphabetCardDetailScreen() {
                 {word_audio}
             </Text>
 
-            <Button testID={`${letter_audio}`} title={'Play Letter'} onPress={playLetter} />
+            {letterAudio && hasAudioLoaded ? (
+                <Button testID={`${letter_audio}`} title={'Play letter'} onPress={playLetter} />
+            ) : (
+                <Text testID="letterAudioError">Error playing `{letter_audio}`</Text>
+            )}
 
-            {wordAudio && hasAudioLoaded && (
+            {wordAudio && hasAudioLoaded ? (
```

Comment:
How do you differentiate between the cases where
- audio loading is still in progress
- audio loading has failed

Reply:
I will start this then rework my test implementation.

Reply:
We have a new implementation.